<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QWidget" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserWindow</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ENTER TEXT</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="QlineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AQpushButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD QUESTION</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AlineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AApushButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD ANSWER</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ENTER TEXT</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
